src/screens/ChatInfoScreen/ChatInfoScreen.tsx

```diff
@@ -292,22 +292,29 @@ const ChatInfoScreen = () => {
               <View style={styles.gap} />
             </>
           }
-          renderItem={({item, index}) => (
-            <View style={styles.parentContact}>
-              <ProfileContact
-                key={index}
-                item={item}
-                onPress={() => onContactPressed(item, params.from)}
-                fullname={item?.name || item?.username}
-                photo={item?.profilePicture}
-                showArrow={handleShowArrow(item)}
-                userId={signedProfileId}
-                ImageComponent={renderImageComponent(item)}
-                isYou={item?.userId === signedProfileId || item?.userId === anonProfileId}
-                from={params?.from}
-              />
-            </View>
-          )}
+          renderItem={({item, index}) => {
+            return (
+              <View style={styles.parentContact}>
+                <ProfileContact
+                  key={index}
+                  item={item}
+                  onPress={() => onContactPressed(item, params.from)}
+                  fullname={item?.name || item?.username}
+                  photo={item?.profilePicture}
+                  showArrow={handleShowArrow(item)}
+                  userId={signedProfileId}
+                  ImageComponent={renderImageComponent(item)}
+                  isYou={
+                    item?.userId === signedProfileId ||
+                    item?.userId === anonProfileId ||
+                    item?.id === signedProfileId ||
+                    item?.id === anonProfileId
+                  }
+                  from={params?.from}
+                />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The logic for determining if a user is the current user has been expanded to check both `userId` and `id` against `signedProfileId` and `anonProfileId`. This could potentially lead to confusion or bugs in the future, as it's not clear why both `userId` and `id` are being checked. If `userId` and `id` can both represent the same concept (the user's ID), it would be better to unify them under one property to avoid confusion.

```diff
- item?.userId === signedProfileId ||
- item?.userId === anonProfileId ||
- item?.id === signedProfileId ||
- item?.id === anonProfileId
+ item?.id === (signedProfileId || anonProfileId)
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->